examples/plot_metrics_with_error.py

```diff
@@ -0,0 +1,176 @@
+# %%
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+================================
+Plotting Metrics with Error Bars
+================================
+"""
+# %%
+# Load and preprocess the data set
+# --------------------------------
+# We download the data set using `fetch_openml` function in `sklearn.datasets`.
+# The original Adult data set can be found at https://archive.ics.uci.edu/ml/datasets/Adult
+# We start by importing the various modules we're going to use:
+#
+import os
+import sys
+nb_dir = os.path.split(os.getcwd())[0]
+if nb_dir not in sys.path:
```

Comment:
why do we need this part?

Reply:
sorry used for local testing, I keep removing it for the PR, then adding it back on the next commit